### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by lipchyk**
Remove dependencies which were introduced in the previous PR
Make use of `github.com/pkg/errors` for errors wrapping and stacktraces

### CODE REVIEW

**Review by AMekss — APPROVED** [review]
LGTM

File: bugsnag.go [hunk 1]
```diff
@@ -12,12 +12,10 @@ import (
 	"reflect"
 	"strconv"
 
-	"github.com/fromatob/bugsnack/error"
-	"github.com/fromatob/bugsnack/hashstruct"
-	"github.com/fromatob/bugsnack/internal/stack"
+	"github.com/pkg/errors"
```
Comment (lucasrenan) [inline bugsnag.go]: excuse me for my ignorance, but where this package is being used exactly?

Comment (gregoriokusowski) [inline bugsnag.go]: They add stacktraces to errors, so here is an example: https://github.com/fromAtoB/bugsnack/pull/3/files#diff-828fe330a0739077fef40a69cc0b721eR112

**Review by gregoriokusowski — APPROVED** [review]
LGTM after removing the panic stuff

**Pull request merged**